app/models/test_track/ab_configuration.rb

```diff
@@ -20,7 +20,7 @@ def variants
 
   def build_variant_hash
     notify_because_ab("configures split with more than 2 variants") if split_variants && split_variants.size > 2
-    { true: true_variant, false: false_variant }
+    { true: true_variant, false: false_variant } # rubocop:disable Lint/BooleanSymbol
```

Comment:
`disable Lint/BooleanSymbol` appears all over the place -- we can just disable it globally, but then we lose the cop in places where we might actually mean to pass a boolean value... 😢 

If only we could add a little bit of type-casting to variant values, like, just as syntactic sugar for all the consuming code... 🤔